finalPrep/modZeroHero.js

```diff
@@ -55,30 +73,56 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - citiesDestroyed (array)
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
+var scaryMonster = {
+  name: 'Bingus',
+  smell: 'stinky',
+  weight: 5000,
+  citiesDestroyed: ['Denver','Thornton','Pueblo'],
+  luckyNumbers: [5,7,85],
+  address: [42, 'ronald', 'CO', 82580]
 
+}
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
-
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name,superpower,age){
+  this.name = name;
+  this.power = superpower;
+  this.age = age;
+  this.archNemesis = 'KennyS';
```

Comment:
It's important to read through instructions carefully - on L92 above, the `archNemesis` property should be assigned to a string `"The Syntax Error"`.